frontend/src/services/api.js

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+
+const API_BASE_URL = 'http://localhost:8000/api'; 
+
+const getAuthToken = () => {
+  return localStorage.getItem('authToken'); 
+};
+
+// Example function to fetch courses
```

Comment:
what do you mean `example function`?

Reply:
I forgot to remove it. I will remove the 'example' word.